src/esp/assets/attributes/ObjectAttributes.cpp

```diff
@@ -17,6 +17,38 @@ const std::map<std::string, esp::assets::AssetType>
         {"suncg", AssetType::SUNCG_SCENE},
 };
 
+const std::string AbstractObjectAttributes::JSONConfigTestString =
```

Comment:
I would expect to find this kind of test data/code in the test files, not in the feature implementation and header files.

Reply:
I completely agree.  I only put this here to act as a kind of reminder, so that any future maintainers would know to expand the tests via this construction when they expand attribute management.